src/main/java/com/google/codeu/data/Datastore.java

```diff
@@ -111,4 +111,31 @@ public void storeMessage(Message message) {
 
     return messages;
   }
+  
+  /** Stores the User in Datastore. */
+  public void storeUser(User user) {
+    Entity userEntity = new Entity("User", user.getEmail());
+    userEntity.setProperty("email", user.getEmail());
+    userEntity.setProperty("aboutMe", user.getAboutMe());
+    datastore.put(userEntity);
+  }
+
+  /**
+   * Returns the User owned by the email address, or null if no matching User was
+   * found.
+   */
+  public User getUser(String email) {
+
+    Query query = new Query("User").setFilter(new Query.FilterPredicate("email", FilterOperator.EQUAL, email));
```

Comment:
Make sure to run the  code formatter on your commit:

https://github.com/google/google-java-format